account_move_name_sequence/views/account_journal.xml

```diff
@@ -14,18 +14,18 @@
         <field name="arch" type="xml">
             <field name="refund_sequence" position="before">
                 <field
-                    name="sequence_id"
-                    required="1"
-                    context="{'default_name': name, 'default_company_id': company_id, 'default_implementation': 'no_gap', 'default_padding': 4, 'default_use_date_range': True, 'default_prefix': (code or 'UNKNOWN') + '/%%(range_year)s/'}"
-                />
+          name="sequence_id"
```

Comment:
It seems your pre-commit configuration is not correct, notice indentation is being screwep up here. I'ts probably taking two spaces instead of four.

Reply:
Not fully applied, there still are changes to the module.

I think we can dismiss this commit for now and include it when targeting the OCA.

Reply:
Not applied yet, precommit changes are still present.

Reply:
The indentation changed after simplify the way to define modifiers as invisible and required. Before, that modifiers were defined as part of attrs and now they are defined individually as was introduced in https://github.com/odoo/odoo/pull/104741.

Reply:
My comment is over the commit that applies pre-commit, which is located before the migration commit.